### ISSUE TITLE
Incompatibility with mobile safari (iOS)

### ISSUE THREAD

**Issue by mike-north** [stefanpenner/ember-browser-stats#8]
Not sure if you were already aware of this, but there is no `window.performance` on mobile safari

https://github.com/stefanpenner/ember-browsery-stats/blob/7c12be10812f8e29ab3dd54102b3ae80942bd944/vendor/memory-stats.js#L53

Not sure if you'd rather investigate a polyfill (if it's even possible to instrument accurately) or maybe just a note in the readme
![screen shot 2015-06-20 at 4 29 24 pm](https://cloud.githubusercontent.com/assets/558005/8269713/9e89a8cc-1769-11e5-9688-6b9a07395d5b.png)

**Comment by stefanpenner** [comment]
I don't think this is something we can polyfil.  It shouldn't crash though, rather gracefully degrade.

**Comment by mike-north** [comment]
agree. I was running into sauce labs failures, which should be avoided if possible

**Comment by stefanpenner** [comment]
seems like a bug upstream: https://github.com/paulirish/memory-stats.js/blob/master/memory-stats.js#L66-L68
